mesa/visualization/templates/js/NetworkModule_d3.js

```diff
@@ -24,6 +24,45 @@ var NetworkModule = function(svg_width, svg_height) {
             g.attr("transform", d3.event.transform);
         }));
 
+    svg.append("svg:defs").selectAll("marker")
```

Comment:
The 2 `svg.append...`'s here are very similar, and so they could be deduplicated to make it clear.

Reply:
There are 3 of them actually.

Reply:
I've deduplicated the shared code so the changes are as short as possible. Also, the network won't be drawn as directed unless "directed": True in portrayal["edges"]. If "directed": False, or not specified, the network will be drawn as undirected.